source/fundamentals/crud/read-operations/skip.txt

```diff
@@ -0,0 +1,82 @@
+=====================
+Skip Returned Results
+=====================
+
+.. default-domain:: mongodb
+
+In this guide, we show you how to skip returned results.
+
+You can skip results using the ``skip()`` method of a 
+:java-docs:`FindIterable </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#skip(int)>`
+instance or by using the :java-docs:`Aggregates.skip() </apidocs/mongodb-driver-core/com/mongodb/client/model/Aggregates.html#sort(org.bson.conversions.Bson)>` 
+static factory method in an aggregation pipeline. 
+
+The ``skip()`` method takes an integer that specifies how many documents
+to omit from the beginning of the list of documents returned by your
+query. 
+
+You can use the ``skip()`` method of a ``FindIterable`` instance as follows:
+
+.. code-block:: java
+
+   collection.find().skip(2); 
+
+The ``Aggregates.skip()`` method implements the ``Bson`` interface to
+specify skip criteria as an argument.  
```

Comment:
1) Since methods don't/can't implement interfaces, we shouldn't make that claim.

2) Usually, when you use a sentence in the structure "___ does Y to do Z", the Y is the precedent for the consequence Z. Is "[implementing the Bson interface]" the direct cause of "[specifying the skip criteria as an argument]"? Maybe you intended to list two different items in which you want to use a conjunction like "and" or "or" instead of a preposition like "to". Let me know if this makes sense.